reporter-influxdb/src/main/java/io/ultrabrew/metrics/reporters/influxdb/InfluxDBReporter.java

```diff
@@ -137,9 +150,14 @@ public InfluxDBReporter build() {
       if (baseUri == null) {
         throw new IllegalArgumentException("Invalid baseUri");
       }
-      if (database == null || database.isEmpty()) {
+      if ((database == null || database.isEmpty()) && 
+          (endpoint == null || endpoint.isEmpty())) {
```

Comment:
You could use the new `isNullOrEmpty()` helper here.

Reply:
Done.